.github/workflows/ami.yml

```diff
@@ -0,0 +1,35 @@
+name: Build AMI
+
+on:
+  push:
+    tags:
+      - 'v*.*.*' # Enforce Semantic Versioning
```

Comment:
have you confirmed what happens if we overwrite a tag? That _is_ something that happens on occasion.